www/about/faq.spt

```diff
@@ -55,6 +55,10 @@ title = _("Frequently Asked Questions")
             '</a>'|safe) }}</p>
     </dd>
 
+    {{ dt(_("Can Liberapay be used to laundry money?"), 'laundry-money') }}
```

Comment:
What we want to do is reassure (potential) users about the legality of Liberapay, so I think the question would be more like "Is Liberapay legal?" or "Does Liberapay respect financial regulations?".